packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -11,12 +11,11 @@ import {ForkChoiceError, ForkChoiceErrorCode, EpochDifference, AncestorStatus} f
 import {ZERO_HASH_HEX} from "../../constants/index.js";
 import {toCheckpointHex} from "../stateCache/index.js";
 import {isOptimisticBlock} from "../../util/forkChoice.js";
-import {ChainEvent} from "../emitter.js";
+import {ChainEvent, ReorgEventData} from "../emitter.js";
 import {REPROCESS_MIN_TIME_TO_NEXT_SLOT_SEC} from "../reprocess.js";
 import {RegenCaller} from "../regen/interface.js";
 import type {BeaconChain} from "../chain.js";
 import {BlockInputType, FullyVerifiedBlock, ImportBlockOpts, AttestationImportOpt} from "./types.js";
-import {PendingEvents} from "./utils/pendingEvents.js";
```

Comment:
- From https://github.com/ChainSafe/lodestar/pull/4973 all remaining events are safe to be emitted in the order of this PR

Reply:
it may be safe from a logic perspective, but we may want to keep this to avoid a future footgun?

Now we will be adding additional work, running all event handlers, interspersed with importBlock, which we want to finish ASAP.

Reply:
It's safe because none of those events trigger side effects, unlike they did before. They are solely consumed by API eventsream clients